migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -49,19 +50,30 @@ The following `imagestreamtags` have been _removed_ from {product-title} 4.2:
 
 include::modules/migration-understanding-cam.adoc[leveloffset=+1]
 
-== Installing the CAM Operator
+== Deploying the Cluster Application Migration tool
 
-You must install the CAM Operator xref:installing-migration-operator-manually_{context}[manually on the {product-title} 3.x source cluster] and xref:installing-migration-operator-with-olm_{context}[with OLM on the {product-title} 4.2 target cluster].
+Deploying the CAM tool involves the following steps:
 
-include::modules/migration-installing-migration-operator-manually.adoc[leveloffset=+2]
-include::modules/migration-installing-migration-operator-olm.adoc[leveloffset=+2]
+. Installing the CAM Operator on all clusters
++
+The CAM Operator installs the following:
+
+* Source cluster(s): Restic and Velero
+* Target cluster: Migration controller, CAM web console, Restic, and Velero
+
+. Configuring cross-origin resource sharing on the source cluster(s) to enable communication with the CAM tool
+
+include::modules/migration-installing-cam-operator-ocp-3.adoc[leveloffset=+2]
+:ocp-3-to-4!:
 
-// == Configuring cross-origin resource sharing
-//
-// You must configure cross-origin resource sharing on the source cluster to enable the CAM tool to communicate with the cluster's API server.
+:context: targetcluster3-4
```

Comment:
I don't know that it's a good idea to change the value of the context attribute here in the middle of an assembly. Why is this being done?

If you need to use an attribute like the `targetcluster3-4` one you created below, you shouldn't need to touch the context here too.

Reply:
Update: I reached out to the rest of the team and they pointed me to the mod docs section that shows that this is what's needed to be done when a module is included multiple times in an assembly. You can ignore my questions on this. Though I do still have questions on a few  attributes that are defined but not ever used.

Reply:
I wanted to use the same module for installing the CAM Operator on 4.1 source clusters and 4.2 target clusters.